content/en/docs/components/model-registry/reference/architecture.md

```diff
@@ -13,24 +13,25 @@ The Model Registry is a passive repository for metadata and is not meant to be a
 {{% /alert %}}
 
 
-Kubeflow Model Registry makes use of the Google community project [ML-Metadata](https://github.com/google/ml-metadata) as one of its core component. ML-Metadata provides a very extensible schema that is generic, similar to a key-value store, but also allows for the creation of logical schemas that can be queried as if they were physical schemas. Those can be manipulated using their bindings in the Python library. This model is extended to provide the metadata management service capabilities for Model Registry.
+Kubeflow Model Registry stores metadata in a backend RDBMS that leverages an extensible ER model inspired by the Google community project [ML-Metadata](https://github.com/google/ml-metadata). This provides a very extensible schema that, while being generic (similar to a key-value store), additionally allows for the creation of logical schemas that can be queried as if they were physical schemas.
```

Comment:
```suggestion
Kubeflow Model Registry stores metadata in a backend RDBMS that leverages an adaptable ER model inspired by the Google community project [ML-Metadata](https://github.com/google/ml-metadata). This provides a very extensible schema that, while being generic (similar to a key-value store), additionally allows for the creation of logical schemas that can be queried as if they were physical schemas.
```

I switched the first `extensible` to `adaptable` to avoid repetition, wdyt?